arrow-data/src/test/kotlin/arrow/data/OptionTest.kt

```diff
@@ -58,6 +60,7 @@ class OptionTest : UnitSpec() {
 
         testLaws(
                 EqLaws.laws(eq(), { genOption(Gen.int()).generate() }),
+                ShowLaws.laws(show(), eq(), { it }),
```

Comment:
You're testing Int here! Use it.some() instead.

Reply:
I always forget this 😅